src/views/Orders.vue

```diff
@@ -266,6 +266,7 @@ export default defineComponent({
       this.currentJob = this.getJob(enumId)
       this.title = title
       this.currentJobStatus = status
+      this.store.dispatch('job/currentJobUpdated', this.currentJob)
```

Comment:
Why to assign job as this.currentJob, we could have local variable and also variable name could be just job
```suggestion
      this.store.dispatch('job/updateCurrentJob', this.currentJob)
```